stairway/src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -722,19 +712,16 @@ private void fillFlightContexts(Connection connection, List<FlightContext> fligh
             flightContext.setFlightStatus(FlightStatus.valueOf(rsflight.getString("status")));
             flightContext.setStepIndex(rsflight.getInt("step_index"));
 
-            // TODO(PF-703): In the current transition away from working_parameters column towards
-            // flightworking table we can either have only JSON or both.  Until we've transitioned,
-            // delegate the decision of which to use to the FlightMap class.
+            // We may have JSON from output_parameters, a set of working parameters from
+            // flightworking table, neither, or both.  Deletgate the decision of which to use to
+            // FlightMap class.
```

Comment:
nit: not sure this comment belongs here any more. Seems like this all about FlightMap's promises and the comments belong on that class.
ditto comment on outputParamsJson

Reply:
I feel like having this in both places doesn't hurt, as it explains why both columns exist in the data model without having to cross-reference `FlightMap`.  That said, if you feel like the docs in `FlightMap` are insufficient I can try to improve them (they look OK to me, but I have the full context in my head).

Reply:
I think it is worth noting here. It would be worth mentioning PF-917 that would remove the output_parameters and working_parameters fields.

Reply:
Docs now reference PF-917.